docker-compose.yml

```diff
@@ -15,17 +15,8 @@ services:
         soft: -1
         hard: -1
 
-  # NOTE: This would be nice to develop with https:// locally, but it doesn't work, for whatever reason, so we need a system caddy instead
```

Comment:
This would never work "easily" (only if the deno piece was also running in docker instead of the host machine), so I'm removing it.